customer-samples/RabbitMQ.Sample/Send/Send.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Datadog.Trace;
+using RabbitMQ.Client;
+
+namespace Send
+{
+    class Send
+    {
+        // This small application follows the RabbitMQ tutorial at https://www.rabbitmq.com/tutorials/tutorial-one-dotnet.html
+        public static void Main(string[] args)
+        {
+            var factory = new ConnectionFactory() { HostName = "localhost" };
+            using (var connection = factory.CreateConnection())
+            {
+                using (var channel = connection.CreateModel())
+                {
+                    channel.QueueDeclare(queue: "hello",
+                                         durable: false,
+                                         exclusive: false,
+                                         autoDelete: false,
+                                         arguments: null);
+
+                    // Start a new Datadog span
+                    using (var scope = Tracer.Instance.StartActive("rabbitmq-publish"))
+                    {
+                        string message = "Hello World!";
+                        var body = Encoding.UTF8.GetBytes(message);
+
+                        // Create BasicProperties and a Headers dictionary to store header information
+                        var properties = channel.CreateBasicProperties();
+                        properties.Headers = new Dictionary<string, object>();
+
+                        // Get properties for the active Datadog span
+                        ulong traceId = scope.Span.TraceId;
+                        ulong spanId = scope.Span.SpanId;
+                        string samplingPriority = scope.Span.GetTag(Tags.SamplingPriority);
+
+                        // Add properties to the Headers dictionary in the following way:
```

Comment:
Thanks for adding sampling priority to the sample. I remembered later that [`SpanContextPropagator`](https://github.com/DataDog/dd-trace-dotnet/blob/ae4feac06f98009d12c9dfa325d66f88c820c919/src/Datadog.Trace/SpanContextPropagator.cs#L47-L60) also injects the `origin` to support Synthetics. Probably less important than sampling priority, but we should consider adding it too for completeness.

Reply:
Looks like there's no public constructor for SpanContext that takes an `origin` so I'm going to leave this as-is